Support.md

```diff
@@ -1,30 +1,67 @@
 # Support Roles
 
+## Junior Support Engineer
+
+_I am starting my journey at Octopus. I am an enthusiastic learner, I show great potential and have a high growth trajectory toward independent contribution._
+
+
+>We avoid distinguishing too much between Junior Support Engineer and Support Engineer. These roles are primarily focused on learning with the aim to grow into the contribution of a Senior Support Engineer or beyond. The primary indicator inside these levels is how far you are along your journey from dependent contribution to independent contribution.
```

Comment:
I don't know if this distinction is helpful or not, but I've always thought of the difference between "junior role" and "role", as a junior being somebody who is actively learning how to do the role and needs support, training, feedback, mentoring, while somebody in a non-junior role also needs those to continue to grow, they don't need it as much to be independently productive.